Plugins: Enable plugin runtime install/uninstall capabilities (#33836)
grafana/grafana: docs/sources/plugins/marketplace.md

+++
title = "Plugin marketplace"
aliases = ["/docs/grafana/latest/plugins/marketplace/"]
weight = 1
+++

> Marketplace for Grafana is a plugin bundled with Grafana versions 8.0+. It allows users to browse and manage plugins from within Grafana. Access is limited to Grafana Admins.

Reviewer: > Access is limited to Grafana Admins

Access to what? The entire app or installing/uninstalling? Can Viewers at least browse plugins?

[screenshot placeholder]

> In order to use the Marketplace for Grafana, it must be enabled in the Grafana [configuration]({{< relref "../administration/configuration.md#marketplace_app_enabled" >}}).

Reviewer: ```suggestion
To use the Marketplace for Grafana, you first need to enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#marketplace_app_enabled" >}}).
```

+++
title = "Plugin marketplace"
aliases = ["/docs/grafana/latest/plugins/marketplace/"]
weight = 1
+++

> 

Author: Do you mean that `docs/sources/plugins/marketplace.md` should match the title here
```
+++
title = "Plugin marketplace"
...
```

IE `docs/sources/plugins/plugin-marketplace.md`◀️ ▶️ `title = "Plugin marketplace"`?

> 

Reviewer 2: Yes.

Marketplace for Grafana is a plugin bundled with Grafana versions 8.0+. It allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the Marketplace.

[screenshot placeholder]

To use the Marketplace for Grafana, you first need to enable it in the Grafana [configuration]({{< relref "../administration/configuration.md#marketplace_app_enabled" >}}).

> ## Install a plugin from the Marketplace

Author: @oddlittlebird Are the docs fine to merge as is, and we can fill in the gaps during the beta?

> ## Install a plugin from the Marketplace

Reviewer 2: Yes, that works
